packages/api/src/services/lsp-state.e2e.ts

```diff
@@ -40,11 +40,25 @@ describe("lsp-state service", function () {
     const service = Service({}, appState);
     assert.ok(service);
   });
+  it("get collateral balance", async function () {
+    const collateralAddress = "0x04Fa0d235C4abf4BcF4787aF4CF447DE572eF828";
+    const service = Service({}, appState);
+    const [address] = registeredContracts;
+    const result = await service.utils.getErc20BalanceOf(collateralAddress, address);
+    assert.ok(result);
+  });
+  it("getPositionCollateral", async function () {
```

Comment:
this was needed to be added to lsp state in order to calculate TVL. explicitly calls erc20 for balance

Reply:
ye, somewhat different than the EMP for this kind of call